lib/Client.js

```diff
@@ -50,6 +50,7 @@ const sleep = (ms) => new Promise((res) => setTimeout(res, ms));
 /**
 * Represents the main Eris client
 * @extends EventEmitter
+* @prop {Object} application Object containing the bot application's ID and its public flags
```

Comment:
the types indicate this should be optional

Reply:
```suggestion
* @prop {Object} [application] Object containing the bot application's ID and it's public flags
```